js/objects-arrays/js-objects-and-arrays_arrays/index.js

```diff
@@ -4,50 +4,67 @@ console.clear();
 // Modify the array exampleArray so that it contains a number and a string.
 
 const exampleArray = [true];
+exampleArray[0] = "string";
```

Comment:
i think you can get the same result in one line with
exampleArray = ["string", 12];